gammapy/image/radial_profile.py

```diff
@@ -0,0 +1,90 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Image utility functions"""
+import numpy as np
+from astropy.coordinates import Angle
+from astropy.wcs.utils import skycoord_to_pixel
+from astropy.table import QTable
+from . import SkyImage
+
+__all__ = [
+    'radial_profile_label_image',
+    'radial_profile',
+]
+
+
+def radial_profile_label_image(image, center, theta_bin=None):
+    """
+   Calculate the labeled image to know which pixel is group together for the radial profile.
+
+   Parameters
+   ----------
+   image : `~gammapy.image.SkyImage`
+         SkyImage of the image we cant to compute the radial profile
+   theta_bin : `~astropy.coordinates.Angle`
+         Bining for the radial profile
+   center : `~astropy.coordinates.SkyCoord`
+         The `SkyCoord` of the pixel used as the center. The default is
```

Comment:
Remove the comment about default for `center`. The default is gone.
Also - put parameter descriptions in the same order as in the function signature, i.e. center before theta_bin.